manifests/m/Maltego/Maltego/4.5.0/Maltego.Maltego.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Maltego.Maltego
+PackageVersion: 4.5.0
+InstallerType: nullsoft
+Dependencies:
+  ExternalDependencies:
+  - Oracle.JDK.21

```

Comment:
```suggestion
```

Remove package dependencies, as we are going to switch to a "JRE-bundled" version.